apps/sumologic/src/app/install/action.tsx

```diff
@@ -0,0 +1,64 @@
+'use server';
+import { logger } from '@elba-security/logger';
+import { z } from 'zod';
+import { SumologicError } from '@/connectors/commons/error';
+import { env } from '@/env';
+import { registerOrganisation } from './service';
+
+const formSchema = z.object({
+  organisationId: z.string().uuid(),
+  accessId: z.string().min(1),
+  accessKey: z.string().min(1),
+  sourceRegion: z.enum(['AU', 'CA', 'DE', 'EU', 'FED', 'IN', 'JP', 'US1', 'US2']),
```

Comment:
Using suggested `sourceRegions`.

```suggestion
  sourceRegion: z.enum(sourceRegions),
```